GETTING_STARTED.md

```diff
@@ -0,0 +1,473 @@
+
+Then, create ssh.ldif with the following contents, being sure to replace the Base DN references with the Base DN of your directory server (ie: dc=example,dc=com):
+
+  ```
+  dn: CN=sshPublicKey,CN=Schema,CN=Configuration,DC=example,DC=com
+  changetype: add
+  objectClass: top
+  objectClass: attributeSchema
+  attributeID: 1.3.6.1.4.1.1466.115.121.1.40
+  cn: sshPublicKey
+  name: sshPublicKey
+  lDAPDisplayName: sshPublicKey
+  description: Users public SSH key
+  attributeSyntax: 2.5.5.5
+  oMSyntax: 22
+  isSingleValued: FALSE
+
+  dn: CN=ldapPublicKey,CN=Schema,CN=Configuration,DC=example,DC=com
+  changetype: add
+  objectClass: top
+  objectClass: classSchema
+  governsID: 1.3.6.1.4.1.1466.115.121.1.40
+  cn: ldapPublicKey
+  name: ldapPublicKey
+  lDAPDisplayName: ldapPublicKey
+  description: Used to store SSH keys in LDAP
+  subClassOf: top
+  objectClassCategory: 3
+  mayContain: sshPublicKey
+  mayContain: uid
+  ```
+
+Next, create sudoers.ldif with SIMILAR contents to the following, being sure to replace the Base DN references with the Base DN of your directory server (ie: dc=example,dc=com). This example LDIF creates a root sudo role and gives *all users* access to that role:
+
+  ```
+  dn: CN=Sudoers,CN=Users,DC=example,DC=com
+  objectClass: top
+  objectClass: container
+  cn: Sudoers
+  name: Sudoers
+  description: Default container for Sudoers configuration
+  distinguishedName: CN=Sudoers,DC=example,DC=com
+
+  # sudo defaults
+  dn: cn=defaults,CN=Sudoers,CN=Users,DC=example,DC=com
+  objectClass: top
+  objectClass: sudoRole
+  cn: defaults
+  description: Default sudoOptions go here
+  sudoOption: env_keep+=SSH_AUTH_SOCK
+
+  dn: CN=root,CN=Sudoers,CN=Users,DC=example,DC=com
+  objectClass: top
+  objectClass: sudoRole
+  cn: root
+  sudoHost: ALL
+  sudoCommand: ALL
+  sudoUser: ALL
+  ```
+
+To load these LDIFS, execute the following, being sure to replace place-holders with proper values:
+
+  ```bash
+  ldbadd -H "ldap://example.com" sudo.ldif --user "<Admin Account Username>" --password "<Admin Account Password>"
+  ldbadd -H "ldap://example.com" ssh.ldif --user "<Admin Account Username>" --password "<Admin Account Password>"
+  ldbadd -H "ldap://example.com" sudoers.ldif --user "<Admin Account Username>" --password "<Admin Account Password>"
+  ```
+
+The next few steps should be done using the Active Directory Users & Computers tool from within Windows. Create at least two users: one that will be used to test that the setup is functioning correctly, and the other that will be used by SSSD to access sudoers data inside LDAP. For example, "test.user" and "sssd.ldap" (used in our example below). You must also grant permission to the ou=Sudoers container that was created via the above LDIF. First, choose "View" and select "Advanced Features". Right click on 'Sudoers' on the left, choose Properties, and then select the Security tab. Allow Authenticated Users read access to the container, and then save your changes.
```

Comment:
We should remove all references to windows servers and use the samba-tool instead.

You may also want to steal some additional code from the samba cookbook that automatically imports the required ldif's rather than requiring the user to manually import them.